### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ArtMot99**

### CODE REVIEW

File: services/movie.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    # If only actors_ids is provided
+    if genres_ids is None and actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    # If both genres_ids and actors_ids are provided
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
```
Comment (ArsenPidhoretskyi) [inline services/movie.py:25]: You can simplify this and write only with 2 conditions:
1. genres_ids is not None
2. actors_ids is not None

File: services/cinema_hall.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from db.models import CinemaHall
+from typing import Any
+
+
+def get_cinema_halls() -> Any:
```
Comment (ArsenPidhoretskyi) [inline services/cinema_hall.py:5]: No, specify the correct type. It does not return Any it return QuerySet

File: services/movie.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+from typing import Any, Optional
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None
+) -> Any:
```
Comment (ArsenPidhoretskyi) [inline services/movie.py:8]: Any is not accepted if you can provide more detailed type

Comment (ArsenPidhoretskyi) [reply services/movie.py:8]: Fix in all places

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
don't work on the master branch - it's a bad practice
https://github.com/mate-academy/py-task-guideline/blob/main/README.md#:~:text=git%20checkout%20%2Db%20develop

**Pull request closed without merging**